src/test/java/org/prebid/server/bidder/zeroclickfraud/ZeroclickfraudBidderTest.java

```diff
@@ -44,11 +42,6 @@ public void setUp() {
         zeroclickfraudBidder = new ZeroclickfraudBidder(ENDPOINT_TEMPLATE, jacksonMapper);
     }
 
-    @Test
-    public void creationShouldFailOnInvalidEndpointUrl() {
-        assertThatIllegalArgumentException().isThrownBy(() -> new DatablocksBidder("invalid_url", jacksonMapper));
```

Comment:
I meant that this test is happy path. 
Add test which will show the real endpoint. (use ENDPOINT_TEMPLATE)